docker-compose.yml

```
@@ -1,7 +1,13 @@
-# Production Docker Compose - uses pre-built images
+# Docker Compose
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)